app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -368,7 +369,7 @@ private void parseFeed() {
      */
     private void beforeShowFeedInformation(Feed feed) {
         final HtmlToPlainText formatter = new HtmlToPlainText();
-        if(Feed.TYPE_ATOM1.equals(feed.getType()) && feed.getDescription() != null) {
+        if (Feed.TYPE_ATOM1.equals(feed.getType()) && feed.getDescription() != null) {
```

Comment:
This and the following lines look like they can also be replaced with your HtmlToPlainText call (without checking feed type). This would also have the advantage that you could remove the call in showFeedInformation and that the fix is also applied to item descriptions, not just podcast descriptions.